### DISCUSSION TITLE
Help with setting up Google Analytics?

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#9065] [Q&A]
### Description

I want to set up Google Analytics to monitor the user activity for my website. I followed the [Quarto documentation](https://quarto.org/docs/websites/website-tools.html#google-analytics) for this as follows:
1. Created a Google Analytics account and set up a data stream. This gave me a Measurement ID: `G-xxxxxxxxx`.
2. I copy pasted this ID in `_quarto.yml` as follows (as suggested in the documentation):
```yml
website:
  google-analytics: "G-xxxxxxxxx"
```

**Observation:** Quarto does inject the gtag code in the generated `.html` files. However, I am not seeing any activity (even Realtime) on my Google Analytics page even after accessing the webpage from various devices:

<img width="1172" alt="Screenshot 2024-03-13 at 11 20 10 PM" src="https://github.com/quarto-dev/quarto-cli/assets/102873841/ca04279b-85f3-4b6e-a7fc-6a047dc4be78">

Upon fresh reloading of my webpage, this is what it looks like in the Network of the Dev tools:

<img width="400" alt="Screenshot 2024-03-13 at 11 24 17 PM" src="https://github.com/quarto-dev/quarto-cli/assets/102873841/6e83c94b-784c-4be1-8d94-e58a36cc54f1">

Does something look awry? Is there a step I didn't follow?

**Comment by noramcgregor** [comment]
I'm having the same issue with mine! Mine is set up as a book so I'm wondering if that's the issue?

**Reply by noramcgregor** [reply]
Ok, for some reason it only worked when I used tracking-id: 

google-analytics:
   tracking-id: "G-XXXXXXX"

rather than just google-analytics: "G-XXXXXXXX"

**Reply by mcanouil** [reply]
hum, thanks for the feedback.
This might be a bug if:

- This does not work
```yaml
website:
  google-analytics: "G-XXXXXXX"
```

- This does work
```yaml
website:
  google-analytics:
    tracking-id: "G-XXXXXXX"
```

Although, I don't see anything obvious:

- https://github.com/quarto-dev/quarto-cli/blob/7bb03a08c8c2af592bc83cd2de39b8ed6c468d74/src/project/types/website/website-analytics.ts#L92-L112

(That's the same code used for Book project)
